manifests/namespace-install.yaml

```diff
@@ -17,6 +28,7 @@ spec:
     - apps
     singular: application
   scope: Namespaced
+  subresources: {}
```

Comment:
Can you explain this new line? What is the affect of it?

Reply:
No idea why it showed up.

I followed https://argoproj.github.io/argo-cd/developer-guide/contributing/#automated-builds-tests

That line is added by `make codegen` - if that line isn't there the CI build fails because the checked-in code doesn't match the new output of codegen.

Reply:
For reference the only changes I made by hand were the 3 lines in pkg/apis/application/v1alpha1/types.go - all other changes in this PR are from running codegen.

Reply:
I think it's safe. The only subresources are scale and status, and neither of those are used in the change.